app/pages/api/quoteEmail.ts

```diff
@@ -0,0 +1,37 @@
+// LIBRARY IMPORTS
+import type { NextApiRequest, NextApiResponse } from "next";
+import nodemailer from "nodemailer";
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  const { senderEmail, subject, message } = req.body;
+
+  // Creates a transporter object using the default SMTP transport
+  const transporter = nodemailer.createTransport({
+    host: process.env.EMAIL_HOST,
+    port: parseInt(process.env.EMAIL_PORT, 10),
+    secure: false,
+    auth: {
+      user: process.env.EMAIL_USER,
+      pass: process.env.EMAIL_PASSWORD,
+    },
+  });
+
+  // Sets up email data
+  const mailOptions = {
+    from: `"Website Contact" <${process.env.EMAIL_USER}>`,
+    to: process.env.EMAIL_TO,
+    subject: `Quote Request from ${senderEmail}: ${subject}`,
+    text: message,
```

Comment:
I think text should be something brief, like "Quote request" rather than the full email message